### ISSUE TITLE
Update?

### ISSUE THREAD

**Issue by bropines** [MustardChef/WSAMagiskDelta#9]
This version hasn't been updated in almost half a year. Can you update it?

**Comment by MustardChef** [comment]
Yes I do plan to continue to update this, with huge updates coming next week.

**Comment by MustardChef** [comment]
> Yes I do plan to continue to update this, with huge updates coming next week.

@bropines The reason why I have not continued to update this repo, is because Magisk Delta has the same issue as Magisk Stable regarding https://github.com/MustardChef/WSABuilds/issues/154. I apologize for my lack of communication regarding the lack of updates.

**Comment by bropines** [comment]
I guess none of the developers of magisk and delta thought about how to fix it.

**Comment by MustardChef** [comment]
> I've come across this too. As I understand it, no one has come yet, thinking how to fix it?

Well, Magisk Alpha seems to be working quite well and has not been caught up in this issue.

Maybe opening a GitHub Issue in the MagiskDelta/ Kitsune Mask GitHub can help shine light on the specifics of what is exactly causing the issue.

**Comment by MustardChef** [comment]
> I've come across this too. As I understand it, no one has come yet, thinking how to fix it?

There is a workaround, but it is quite finicky. You can find it by going to https://github.com/MustardChef/WSABuilds/issues/154, and scrolling a bit down.

**Comment by bropines** [comment]
> > I've come across this too. As I understand it, no one has come yet, thinking how to fix it?
> 
> There is a workaround, but it is quite finicky. You can find it by going to [MustardChef/WSABuilds#154](https://github.com/MustardChef/WSABuilds/issues/154), and scrolling a bit down.

I use it in ordinary magic. I wrote a script to replace folders, and I run it every time I install something.

**Comment by MustardChef** [comment]
> > > I've come across this too. As I understand it, no one has come yet, thinking how to fix it?
> > 
> > There is a workaround, but it is quite finicky. You can find it by going to [MustardChef/WSABuilds#154](https://github.com/MustardChef/WSABuilds/issues/154), and scrolling a bit down.
> 
> I use it in ordinary magic. I wrote a script to replace folders, and I run it every time I install something.

I will create a repo for WSA Builds with Magisk Alpha, which should not have any issues with module installations.

**Comment by bropines** [comment]
> > > > I've come across this too. As I understand it, no one has come yet, thinking how to fix it?
> > > 
> > > 
> > > There is a workaround, but it is quite finicky. You can find it by going to [MustardChef/WSABuilds#154](https://github.com/MustardChef/WSABuilds/issues/154), and scrolling a bit down.
> > 
> > 
> > I use it in ordinary magic. I wrote a script to replace folders, and I run it every time I install something.
> 
> I will create a repo for WSA Builds with Magisk Alpha, which should not have any issues with module installations.

Okay, thanks. I also wrote in the magisk delta repository. Let's see if he can fix it.
